rke/k8s_rke_system_images.go

```diff
@@ -7142,6 +7142,92 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
+			Etcd:                      "rancher/mirrored-coreos-etcd:v3.4.16-rancher1",
+			Kubernetes:                "rancher/hyperkube:v1.21.12-rancher1",
+			Alpine:                    "rancher/rke-tools:v0.1.80",
+			NginxProxy:                "rancher/rke-tools:v0.1.80",
+			CertDownloader:            "rancher/rke-tools:v0.1.80",
+			KubernetesServicesSidecar: "rancher/rke-tools:v0.1.80",
+			KubeDNS:                   "rancher/mirrored-k8s-dns-kube-dns:1.17.4",
+			DNSmasq:                   "rancher/mirrored-k8s-dns-dnsmasq-nanny:1.17.4",
+			KubeDNSSidecar:            "rancher/mirrored-k8s-dns-sidecar:1.17.4",
+			KubeDNSAutoscaler:         "rancher/mirrored-cluster-proportional-autoscaler:1.8.3",
+			Flannel:                   "rancher/mirrored-coreos-flannel:v0.15.1",
+			FlannelCNI:                "rancher/flannel-cni:v0.3.0-rancher6",
+			CalicoNode:                "rancher/mirrored-calico-node:v3.19.2",
+			CalicoCNI:                 "rancher/mirrored-calico-cni:v3.19.2",
+			CalicoControllers:         "rancher/mirrored-calico-kube-controllers:v3.19.2",
+			CalicoCtl:                 "rancher/mirrored-calico-ctl:v3.19.2",
+			CalicoFlexVol:             "rancher/mirrored-calico-pod2daemon-flexvol:v3.19.2",
+			CanalNode:                 "rancher/mirrored-calico-node:v3.19.2",
+			CanalCNI:                  "rancher/mirrored-calico-cni:v3.19.2",
+			CanalControllers:          "rancher/mirrored-calico-kube-controllers:v3.19.2",
+			CanalFlannel:              "rancher/mirrored-coreos-flannel:v0.15.1",
+			CanalFlexVol:              "rancher/mirrored-calico-pod2daemon-flexvol:v3.19.2",
+			WeaveNode:                 "weaveworks/weave-kube:2.8.1",
+			WeaveCNI:                  "weaveworks/weave-npc:2.8.1",
+			AciCniDeployContainer:     "noiro/cnideploy:5.2.3.2",
+			AciHostContainer:          "noiro/aci-containers-host:5.2.3.2",
+			AciOpflexContainer:        "noiro/opflex:5.2.3.2",
+			AciMcastContainer:         "noiro/opflex:5.2.3.2",
+			AciOpenvSwitchContainer:   "noiro/openvswitch:5.2.3.2",
+			AciControllerContainer:    "noiro/aci-containers-controller:5.2.3.2",
+			AciGbpServerContainer:     "noiro/gbp-server:5.2.3.2",
+			AciOpflexServerContainer:  "noiro/opflex-server:5.2.3.2",
+			PodInfraContainer:         "rancher/mirrored-pause:3.6",
+			Ingress:                   "rancher/nginx-ingress-controller:nginx-1.2.0-rancher1",
+			IngressBackend:            "rancher/mirrored-nginx-ingress-controller-defaultbackend:1.5-rancher1",
+			IngressWebhook:            "rancher/mirrored-ingress-nginx-kube-webhook-certgen:v1.1.1",
+			MetricsServer:             "rancher/mirrored-metrics-server:v0.5.0",
+			CoreDNS:                   "rancher/mirrored-coredns-coredns:1.8.4",
+			CoreDNSAutoscaler:         "rancher/mirrored-cluster-proportional-autoscaler:1.8.3",
+			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.6",
+			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.18.0",
+		},
+		"v1.22.9-rancher1-2": {
```

Comment:
New 1.22 version needs to be defined further down after the latest already defined 1.22 - we want them to be sorted.